python/ql/src/experimental/Security-old-dataflow/CWE-601/UrlRedirect.ql

```diff
@@ -3,6 +3,8 @@
  * @description URL redirection based on unvalidated user input
  *              may cause redirection to malicious web sites.
  * @kind path-problem
+ * @problem.severity error
+ * @id py/old/url-redirection
```

Comment:
To be honest I'd be happier to just delete these queries, as they are becoming a maintenance burden with no benefit.

Reply:
While I agree that we should do this soon, I don't really make that decision a part of this PR (as explained above)